crpc/go/server.go

```diff
@@ -453,3 +455,42 @@ func (s *Server) PaymentReceived(_ context.Context,
 
 	return resp, nil
 }
+
+//
+// AccountBalance is used to determine account balance for requested asset
```

Comment:
Shouldn't we put in the comment that pending balance is also included? Because external systems might use account balance as if funds already available, but they are not.

Reply:
fixed